LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2StrategyElement_LWObjectives.uc

```diff
@@ -262,7 +262,7 @@ static function CreateBlacksiteMission_LW(XComGameState NewGameState, XComGameSt
 	RewardTemplate = X2RewardTemplate(StratMgr.FindStrategyElementTemplate('Reward_None')); // no rewards for completing story objectives
 	RewardState = RewardTemplate.CreateInstanceFromTemplate(NewGameState);
 	Rewards.AddItem(RewardState);
-	MissionState = CreateMission(NewGameState, Rewards, 'MissionSource_BlackSite', 8); // as far away from player as possible, because reasons
+	MissionState = CreateMission(NewGameState, Rewards, 'MissionSource_BlackSite', eGameRule_PlayerTurnEnd); // as far away from player as possible, because reasons
```

Comment:
Oops 😄  I'm glad I just changed this code today, otherwise I wouldn't have noticed!

Reply:
Embarassing, sorry. That was me taking a Ctrl-F shortcut to find more of these per cycle because uc-analyzer stops after the first type error.